content/home/usecases/data-tabs/tab2.md

```diff
@@ -2,7 +2,7 @@
 title : "usecase-tab"
 name : media
 heading : Media Content Storage
-desc : Store large volumes of video, photo, or audio media, including tape / physical alternatives.
+desc : Store large volumes of video, photo, or audio media, including tape/physical alternatives.
```

Comment:
what is tape/physical alternatives mean? just curious

Reply:
I have no idea lol but this was reviewed by Taylor today

Reply:
lol